sources/api/static-pods/src/main.rs

```diff
@@ -0,0 +1,16 @@
+#![deny(rust_2018_idioms)]
+
+#[cfg(k8s_variant)]
+mod static_pods;
+#[cfg(k8s_variant)]
+#[macro_use]
+extern crate log;
+
+#[cfg(k8s_variant)]
+#[tokio::main]
+async fn main() {
+    static_pods::main().await
+}
+
+#[cfg(not(k8s_variant))]
```

Comment:
Not that it hurts anything, but if we are only going to include this binary in k8s variants, do we need the extra protection of conditional compilation?

Reply:
See my response here: https://github.com/bottlerocket-os/bottlerocket/pull/1317#discussion_r575451214